### ISSUE TITLE
FTBFS on Arch Linux x86_64

### ISSUE THREAD

**Issue by heftig** [joncampbell123/dosbox-x#184]
Had to do this in order to get it to compile with g++ 5.3.0 (otherwise `isinf` is not declared):

``` diff
diff --git i/src/fpu/fpu_instructions.h w/src/fpu/fpu_instructions.h
index 33d1359..b3f435e 100644
--- i/src/fpu/fpu_instructions.h
+++ w/src/fpu/fpu_instructions.h
@@ -281,6 +281,7 @@ static void FPU_FBST(PhysPt addr) {
 #else
 # include <math.h>
 # include <cmath>
+# define isinf(x) (std::isinf(x))
 # define isdenormal(x) (!std::isnormal(x))
 #endif
```

**Comment by neuromancer** [comment]
This is no longer needed with g++ 7.1.1. I had to delete your patch [in the AUR package](https://aur.archlinux.org/packages/dosbox-x-git/) otherwise it will fail to compile.

**Comment by heftig** [comment]
Not my AUR package, FWIW.

This has been fixed in d7bca016548e44dc46b5036f7a00fe81cda7d86a.
